### ISSUE TITLE
Automatic session for firefox container tabs

### ISSUE THREAD

**Issue by MrGreenTea** [kagisearch/browser_extensions#75]
I use [firefox account](https://addons.mozilla.org/en-US/firefox/addon/multi-account-containers/) containers a lot.
When I try to search from such a tab I have to login again to kagi.com
With this extension it would be great if it could do something similar to private mode and use my saved kagi login in these container tabs too.

**Comment by Sol-Mane** [comment]
Seconding this feature request; in theory this would just re-use the `token` mechanism used for private browsing.

Is it possible to fund the development of this feature? :-)

**Comment by vprelovac** [comment]
Noted. If anyone wants to give it a try, we accept pull requests.

**Comment by dix** [comment]
I'm also a daily user of _Firefox Multi-Account Containers_ and its friend [Temporary Containers](https://addons.mozilla.org/en-US/firefox/addon/temporary-containers/ "Mozilla").

If the extension is configured correctly (giving all the rights in kagi.com and having a login token), everything should work as expected.

You'll probably need to reinstall the extension to force it prompting you for the authorizations.

Basically : 
1. Login to kagi.com on a default tab/container
2. Install the Kagi extension
3. Go to kagi.com on a default tab/container
4. Click on the Kagi extension icon, you should be asked to set it up and to give the authorizations on the domain
5. Open another container tab, try to do a search, it should work

**Comment by MrGreenTea** [comment]
@dix Thank you for the step-by-step. I could reproduce it and now it works for me.
@kagisearch I would be fine with closing this issue as it works like expected (after reinstalling the extension)
